### DISCUSSION TITLE
whisper-v3 returns punctuaiton sometimes and sometimes not?

### DISCUSSION THREAD

**Discussion by silvacarl2** [openai/whisper#2065] [Q&A]
we noticed that whisper-v3 returns punctuaiton sometimes and sometimes not, and sometiems capitlziaes in the right places and sometimes it is all lower case.

any idea about why this would be?

**Comment by glangford** [comment]
See this discussion. `large-v2` seems to be more reliable.

* https://github.com/openai/whisper/discussions/1762#discussioncomment-7559886

**Comment by silvacarl2** [comment]
thx!  i missed that!

**Comment by silvacarl2** [comment]
https://deepgram.com/learn/whisper-v3-results
